src/_buttons.scss

```diff
@@ -258,10 +258,41 @@
   }
 }
 
+.circle-button--white-bg {
```

Comment:
Can we call this `circle-button--white`? (And I think I'll rename `circle-button--light` to `circle-button--transparent`.)